README.md

```diff
@@ -1,138 +1,70 @@
+
+# Automations
+
+AutoDuty supports a wide range of automations and integrations with other plugins. Below are just some of what AD can offer while automating instanced content:
+
+| Before runs | During the run | Between runs | After the runs |
+| -------- | -------- | -------- | -------- |
+| Move to an inn, house, or FC house <br /> Auto self- or NPC- repair <br /> Execute commands (i.e. SND scripts) <br /> Auto consume items | Loot treasure coffers <br /> Manage plugin states | Auto extract materia <br /> Auto desynth loot <br /> Auto turn in loot to GC <br /> Use AutoRetainer <br /> Auto equip best gear | Stop looping at a level <br /> Stop looping when you run out of rested XP <br /> Turn on AutoRetainer multi mode <br /> Shut down your computer |
+
+<hr />
+
+# Installation
+
+Add `https://puni.sh/api/repository/herc` to your plugin repositories and then search for `AutoDuty` in the Plugin Installer to install AutoDuty.
+
+Settings can be accessed via the Plugin Installer or using the chat command `/ad`.
+
+## Required Plugins
+
+Additionally, the following plugins are required. Right-click the name of the plugin and copy the URL to use it to add to your repositories in-game.
+
+- [vnavmesh](https://puni.sh/api/repository/veyn): Automatic navigation with waypoints
+- [Rotation Solver Reborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic rotation execution, all jobs supported
+- [Veyn's Boss Mod](https://puni.sh/api/repository/veyn) or [BossmodReborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic fight execution for bosses
+
+## Optional Plugins
+
+The following plugins are optional. They integrate with AutoDuty well and in some cases can be triggered by AD itself.
+
+- [Deliveroo](https://plugins.carvel.li): Automatically turns in items to your Grand Company
+- [AutoRetainer](https://love.puni.sh/ment.json): Automatically manages your retainers and submersibles as well as sends them out on ventures
+
+# Getting help
+
+When you've found a bug or think you have some issue with the plugin, please ask in [Discord](https://discord.gg/punishxiv). It might be a known issue or people might be able to help you quickly. Additionally, sometimes it might be an issue with one of the required plugins. Users in Discord will be able to help triage the issue and send you to the right place.
+
+Best practice is to not say "I died in this dungeon and I don't know why." Please make sure to include as much detail as possible. What boss were you on? Did you get stuck in a specific spot?
+
+For support with Veyn's Boss Mod and vnavmesh, please ask in the [Puni.sh Discord server](https://discord.gg/punishxiv). For support with BossModReborn and Rotation Solver Reborn, please ask in the [Combat Reborn Discord server](https://discord.gg/p54TZMPnC9).
```

Comment:
these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725
vbm -> https://discord.com/channels/1001823907193552978/1191076246860349450

Reply:
Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)